stdlib/3/compileall.pyi

```diff
@@ -34,11 +40,29 @@ if sys.version_info >= (3, 7):
         legacy: bool = ...,
         optimize: int = ...,
         invalidation_mode: Optional[PycInvalidationMode] = ...,
+        *,
+        stripdir: Optional[_Path] = ...,
```

Comment:
This must actually be a string because it does `stripdir.split(os.path.sep)`

Reply:
Great catch, contradicts the documentation. I filed https://bugs.python.org/issue40447

Reply:
Since the bug probably won't be fixed very soon, can you just change the stub for now?